de.metas.fresh/de.metas.fresh.base/src/main/java/de/metas/fresh/picking/form/FreshPackingItemHelper.java

```diff
@@ -53,20 +59,24 @@ public static IFreshPackingItem cast(final IPackingItem item)
 		return (IFreshPackingItem)item;
 	}
 
+	/**
+	 * Invokes {@link #cast(IPackingItem)} with the given <code>item</code>, then returns the result of {@link IFreshPackingItem#copy()}.
+	 *
+	 * @param item
+	 * @return
+	 */
 	public static IFreshPackingItem copy(final IPackingItem item)
 	{
-		if (item instanceof IFreshPackingItem)
```

Comment:
@teosarca can you pls check out if this is change of mine is OK? maybe you had a reason for not invoking cast()?